src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -35,6 +35,12 @@ import ContentListFilters from './ContentListFilters';
 import Hide from '../Hide/Hide';
 import EmptyTableState from './components/EmptyTableState';
 import { useQueryClient } from 'react-query';
+import StatusIcon from './components/StatusIcon';
+
+export const red = global_danger_color_100.value;
+export const green = global_success_color_100.value;
+export const yellow = '#F0AB00';
+export const brown = '#795600';
```

Comment:
I wanted to put all the styling stuff in `StatusIcon.tsx`, but calling createUseStyles there gives an error: "React has detected a change in the order of Hooks called by..."